popup-display/CustomerData/PopupCartData.php

```diff
@@ -21,13 +21,16 @@
 use Magento\Framework\Pricing\Helper\Data as PricingHelper;
 use Magento\Sales\Model\ResourceModel\Report\Bestsellers\CollectionFactory as BestSellersCollectionFactory;
 use Prestafy\PopupDisplay\Helper\Data as Helper;
+use Magento\Checkout\Model\Session as CheckoutSession;
+use Magento\Catalog\Api\ProductRepositoryInterface as ProductRepositoryInterface;
 
 /**
  * PopupCart source
  */
 class PopupCartData implements SectionSourceInterface
 {
-    const CONFIG_PRODUCT_LIMIT = 4;
+    //const CONFIG_PRODUCT_LIMIT = 4;
+    const CONFIG_PRODUCT_LIMIT = 2;
```

Comment:
Hi, thank you for your contribution. Why did you change this config to 2?

Reply:
Hello, that's only for my purpose ;-) . You should rechange it to .